src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -54,6 +55,8 @@ public CommandResult execute(Model model) throws CommandException {
 
             model.setStudents(studentsToSort);
 
+            return new CommandResult(String.format(MESSAGE_SUCCESS, "students", sortBy, directionOfSort));
```

Comment:
Would extracting this out into a static final variable be better? So that your tests can use the same string as well.

Reply:
Same as the comment below